clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/KafkaStoreIngestionService.java

```diff
@@ -18,6 +18,7 @@
 import static java.lang.Thread.currentThread;
 import static java.lang.Thread.sleep;
 
+import com.google.common.collect.ImmutableMap;
```

Comment:
I would prefer that we don't pull in guava classes. They tend to be unstable across versions and cause compatibility headaches.

Reply:
Got it. It has been merged but will fix it in my next change.